docs/sdk/core-kit/sfa-android/usage.mdx

```diff
@@ -142,7 +127,7 @@ class MainActivity : AppCompatActivity() {
   private fun signIn() {
     val jwt = JWT(idToken) // idToken is the JWT token obtained from auth provider or your custom JWT provider
     val sub = jwt.getClaim("sub").asString() // get sub claims
-    loginParams = LoginParams("web3auth-firebase-examples", "$sub", "$idToken")
+    loginParams = LoginParams("w3a-firebase-demo", "$sub", "$idToken")
     try {
         torusKey = singleFactorAuth.getKey(loginParams).get()
```

Comment:
Here, torusKey is the privateKey, so it's better to be direct for a developer who doesn't know what is a torusKey. torusKey is an internal name.